mm/dynamic/c/Magazine.h

```
@@ -0,0 +1,119 @@
+typedef struct Depot {
+    RustSpinLock* lock;
+    SizeClassDepot size_class_depots[NUM_SIZE_CLASSES];
+} Depot;
+
+
+// =================================================================================================
+// Global Variables
+// =================================================================================================
+
+// The array of per-CPU caches. Indexed by CPU ID.
+extern PerCpuCache per_cpu_caches[MAX_CPU_CORES];
```

Clang (14.0.6)

[warning] variable 'per_cpu_caches' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)